src/Raven.Server/Documents/Patch/ScriptRunner.cs

```diff
@@ -293,39 +292,65 @@ private void FillDoubleArrayFromJsArray(double[] array, ArrayInstance jsArray, s
                     ++i;
                 }
             }
-            
-            private JsValue AppendTimeSeries(JsValue self, JsValue[] args)
+
+            private JsValue TimeSeries(JsValue self, JsValue[] args)
+            {
+                AssertValidDatabaseContext("timeseries(doc, name)");
+
+                var originalArgs = new ArrayInstance(ScriptEngine);
+                originalArgs.FastAddProperty("length", 0, true, false, false);
+                ScriptEngine.Array.PrototypeObject.Push(originalArgs, args);
```

Comment:
Why do you need this? Why not just use the `args` ?

Reply:
I also didn't think that this was needed, and I originaly just used the `args` here

but then I noticed that I had a single test that was always failing :
`CanAppendTimeSeriesByPatch_WithoutTag`
with InlineData : `@"timeseries(id(this), args.timeseries).append(new Date(args.timestamp), args.values);"`
(the 2nd inline data for this test, with explicit tag = null, worked fine) :
https://github.com/ravendb/ravendb/blob/v5.0/test/SlowTests/Client/TimeSeries/Patch/TimeSeriesPatchTests.cs#L174

and in this failing test, when I reached line 305 here: https://github.com/ravendb/ravendb/pull/10387/files#diff-d69adfc2e1af26dc65c54a25d0723462R305 
`AppendTimeSeries(args[0], args[1], values));`
args[0] was the timestamp and args[1] was the values (i.e at this point `args` were the arguments for `AppendTimeSeries`, not the arguments for `TimeSeries`)
I tried to investigate that, but couldn't figure out why this happned, and why only in this specific case..

cloning the `args` into `originalArgs` solved this problem

Reply:
Okay, the issue is that Jint is reusing instances. Instead of allocating a new instance of the `args`, it will reuse it.
I changed things so the value is carried in the object itself.

Reply:
see https://github.com/ravendb/ravendb/pull/10404